src/SortChain.php

```diff
@@ -42,6 +48,10 @@ public function keys($collection)
 
     public function __invoke($a, $b)
     {
+        if (empty($this->callables)) {
+            return static::$nullOperatorEquality;
```

Comment:
That said, if you have a list of completely identical elements, does it really matter which order they get sorted? :grinning: